django/contrib/admin/templates/admin/base.html

```diff
@@ -5,8 +5,8 @@
 <title>{% block title %}{% endblock %}</title>
 <link rel="stylesheet" type="text/css" href="{% block stylesheet %}{% static "admin/css/base.css" %}{% endblock %}">
 {% if not is_popup and is_nav_sidebar_enabled %}
-  <link rel="stylesheet" type="text/css" href="{% static "admin/css/nav_sidebar.css" %}">
-  <script src="{% static 'admin/js/nav_sidebar.js' %}" defer></script>
+  <link rel="stylesheet" type="text/css" href="{% block nav_sidebar_stylesheet %}{% static "admin/css/nav_sidebar.css" %}{% endblock %}">
+  <script src="{% block nav_sidebar_script %}{% static 'admin/js/nav_sidebar.js' %}{% endblock %}" defer></script>
```

Comment:
There's no need for blocks here. You configure a project level static folder as normal and override the provided files.

Reply:
In `base.html` All `styles` and `scripts` can be customized through block, except this one